GW2EIEvtcParser/EncounterLogic/Raids/W8/DecimaTheStormsinger.cs

```diff
@@ -70,6 +71,57 @@ public DecimaTheStormsinger(int triggerID) : base(triggerID)
             new PlayerDstHitMechanic(Sparkwave, "Sparkwave", new MechanicPlotlySetting(Symbols.TriangleDown, Colors.LightOrange), "Sparkwave.H", "Hit by Sparkwave (Transcendent Boulders Cone)", "Sparkwave Hit", 0),
             new PlayerDstHitMechanic(ChargedGround, "Charged Ground", new MechanicPlotlySetting(Symbols.CircleOpenDot, Colors.CobaltBlue), "CharGrnd.H", "Hit by Charged Ground (Transcendent Boulders AoEs)", "Charged Ground Hit", 0),
 
+            new PlayerDstHitMechanic([FulgentFenceCM, FluxlanceFusilladeCM, FluxlanceSalvoCM1, FluxlanceSalvoCM2, FluxlanceSalvoCM3, FluxlanceSalvoCM4, FluxlanceSalvoCM5, ChorusOfThunderCM, DiscordantThunderCM], "This Bug Can Dance", new MechanicPlotlySetting(Symbols.Pentagon, Colors.Lime), "BugDance.Achiv", "Achievement Elibigility: This Bug Can Dance", "Achiv: This Bug Can Dance", 0).UsingChecker((adhe, log) =>
+            {
+                // Only available in CM
+                if (!isCM)
```

Comment:
use UsingEnable API for this check, instead of checking for every candidate event.